### PR TITLE
Update all languages except Java

### PR DESCRIPTION
**Pull request by basta-stripe**
Some big changes. I started on the Python version because I love Python and it needed some love.

1. `failure.html` shouldn't even exist. `failure_url` should create a new account link and immediately redirect the user.
2. `success_url` doesn't imply that the onboarding completed, it implies that the user returned successfully. I changed the wording of `success.html`
3. Converted from `requirements.txt` to `Pipfile`. It's safer and much easier for devs (and is consistent between *nix and Windows)
4. Bumped werkzeug to 0.15.5 for Python 3.8.* compat

If this looks good I can probably get the PHP and Node versions updated, but might need some help with the Java version.

### CODE REVIEW

**Comment by auchenberg-stripe** [comment]
LGTM

File: server/python/server.py [hunk 1]
```diff
@@ -34,20 +39,38 @@ def onboard_user():
...
+
+
+def _generate_account_link(account_id, origin):
     account_link = stripe.AccountLink.create(
         type='onboarding',
-        account = account.id,
-        failure_url = '{}/failure.html'.format(return_url),
-        success_url = '{}/success.html'.format(return_url)
+        account=account_id,
+        failure_url=f'{origin}/onboard-user/refresh',
```
Comment (auchenberg-stripe) [inline server/python/server.py]: Couldn't this end up in an infinite loop if someone goes wrong during the on-boarding process?

Comment (basta-stripe) [reply server/python/server.py]: No. The error conditions that can cause the user to end up at `failure_url` also cause the account link create call to 400.

**Comment by basta-stripe** [comment]
@auchenberg-stripe Would you mind taking another look over? I updated Node and PHP as discussed; though I don't have a way to test the PHP version.

**Pull request merged**